src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -12564,5 +12598,25 @@ .maxstack  6
 }
 ");
         }
+
+
+        [Theory, WorkItem(55609, "https://github.com/dotnet/roslyn/issues/55609")]
+        [InlineData(@"$""{h1}{h2}""")]
+        [InlineData(@"$""{h1}"" + $""{h2}""")]
+        public void RefStructHandler_DynamicInHole(string expression)
+        {
+            var code = @"
+dynamic h1 = 1;
+dynamic h2 = 2;
+CustomHandler c = " + expression + ";";
+
+            var handler = GetInterpolatedStringCustomHandlerType("CustomHandler", "ref struct", useBoolReturns: false);
+
+            var comp = CreateCompilationWithCSharp(new[] { code, handler });
+
+            // Note: We don't give any errors when mixing dynamic and ref structs today. If that ever changes, we should get an
```

Comment:
Do we have an issue tracking that?

Reply:
Not as far as I'm aware, @jaredpar do you know if we have one?

Reply:
No we don't. I was [speculating](https://github.com/dotnet/roslyn/issues/55609#issuecomment-898646888) on the issue here that we should consider adding one in a warning wave. Guessing not a lot of people try this (it's basically two features at odds) but when you do it would be nice to be warned about it.